src/documentation/common/CodeEmbed.tsx

```diff
@@ -180,7 +172,7 @@ const CodeEmbed = ({
           _focusVisible={{
             outline: "none",
           }}
-          onKeyDown={handleKeyDown}
+          // onKeyDown={handleKeyDown}
```

Comment:
What's intended here?

Reply:
A failure to fully remove the previous way of registering the relevant keydown handler.

Reply:
Removed here [e313a04](https://github.com/microbit-foundation/python-editor-v3/pull/1175/commits/e313a0484bb360d53342cf05b58c8de00cbe65c2).